cypress/integration/my-orcid/my-orcid-top-bar.e2e.js

```diff
@@ -31,39 +34,117 @@ describe('My Orcid top bar' + runInfo(), () => {
     it('display `Display your iD on other sites` button', () => {})
   })
   describe('Names' + runInfo(), () => {
-    it('display user published name if exists"', () => {})
-    it('display user given names and family names"', () => {})
-    it('display multiple other names if exists with privacy"', () => {})
-
-    describe('Names edit modals' + runInfo(), () => {
-      it('edit user given names"', () => {
-        // Expect changes to be display outside and inside of the modal
-      })
-      it('edit user family names"', () => {
-        // Expect changes to be display outside and inside of the modal
-      })
-      it('edit user published name"', () => {
-        // Expect changes to be display outside and inside of the modal
-      })
-      it('edit user names privacy"', () => {
-        // Expect changes to be display outside and inside of the modal
-      })
-      it('add other names with default privacy', () => {
-        // Expect changes to be display outside and inside of the modal
-      })
-      it('remove/delete other names', () => {
-        // Expect changes to be display outside and inside of the modal
-      })
-      it('Drag and drop to rearrange', () => {
-        // Expect changes to be display outside and inside of the modal
-      })
-      it('change privacy', () => {
-        // Expect changes to be display outside and inside of the modal
-      })
-      it('make changes and cancel', () => {
-        // Expect changes NOT to be display outside and inside of the modal
+    before(() => {
```

Comment:
Please do not delete the test description that we are not implementing. 
This is meant to be developed later by someone else.

Reply:
like `change privacy` or `make changes and cancel`